### DISCUSSION TITLE
Applying styles to hook classes - advice please

### DISCUSSION THREAD

**Discussion by wickham-c** [filamentphp/filament#12276] [Help] [answered]
### Package

Infolist builder

### Package Version

v3.2.52

### How can we help you?

For an Infolist, I would like to create a little more contrast between the background colour of a Section header and the background colour of the same Section's content. I've read [this advice](https://filamentphp.com/docs/3.x/support/style-customization#applying-styles-to-hook-classes) on applying styles to hook classes and have come up with the following styles...

```
.fi-section-header {
    background-color: rgba(var(--gray-100),1);
}
.dark .fi-section-header {
    background-color: rgba(var(--gray-800),1)
}
```
I believe I need to add the above styles to the following file: `public\css\filament\filament\app.css`.

This raises two questions in my mind...

1. Is the above CSS file the recommended location for me to add my CSS styles (e.g. I'm presuming this file won't be overwritten by future Filament releases). If not, where should I add them?
2. Will the above styles overwrite existing rules for all properties of the `.fi-section-header` class selector? If so, how would I append this `background-color` property rule whilst retaining existing rules for all other properties for this selector?

Your advice would be much appreciated.

**Comment by leandrocfe** [comment, marked as answer]
If you are using only the Infolist builder, you should add it in the app.css
https://filamentphp.com/docs/3.x/infolists/installation#configuring-styles

If you are using a Panel builder, you should customize a theme
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme

You can use these classes or create new classes and use in components with `->extraAttributes(['class' => 'new-class'])`
https://filamentphp.com/docs/3.x/infolists/entries/getting-started#custom-attributes

**Reply by wickham-c** [reply]
Many thanks @leandrocfe 

I'm using Panel builder, so I've gone through the steps for creating a custom theme and have added the CSS rules I shared above to the `resources\css\filament\app\theme.css` file (which was generated using the steps).
